src/pmdas/denki/denki.c

```diff
@@ -412,20 +452,20 @@ static pmLongOptions longopts[] = {
     PMDA_OPTIONS_END
 };
 static pmdaOptions opts = {
-    .short_options = "D:d:i:l:r:pu:U:6:?",
+    .short_options = "C:D:d:i:l:r:pu:U:6:?",
```

Comment:
This -C option is not needed - the -D option already supports what you seek here.